src/dlgProfilePreferences.cpp

```diff
@@ -88,129 +86,127 @@ dlgProfilePreferences::dlgProfilePreferences( QWidget * pF, Host * pH )
 
     QDir dir(path);
 
-    QStringList entries = dir.entryList( QDir::Files, QDir::Time );
+    QStringList entries = dir.entryList(QDir::Files, QDir::Time);
     QRegExp rex("\\.dic$");
-    entries = entries.filter( rex );
-    for( int i=0; i<entries.size(); i++ )
-    {
-        QString n = entries[i].replace( ".dic", "" );
-        auto item = new QListWidgetItem( entries[i] );
-        dictList->addItem( item );
-        if( entries[i] == mpHost->mSpellDic )
-        {
-            item->setSelected( true );
+    entries = entries.filter(rex);
+    for (int i = 0; i < entries.size(); i++) {
+        QString n = entries[i].replace(".dic", "");
+        auto item = new QListWidgetItem(entries[i]);
+        dictList->addItem(item);
+        if (entries[i] == mpHost->mSpellDic) {
+            item->setSelected(true);
         }
     }
 
-    if( pH->mUrl.contains( QStringLiteral( "achaea.com" ), Qt::CaseInsensitive )
-     || pH->mUrl.contains( QStringLiteral( "aetolia.com" ), Qt::CaseInsensitive )
-     || pH->mUrl.contains( QStringLiteral( "imperian.com" ), Qt::CaseInsensitive )
-     || pH->mUrl.contains( QStringLiteral( "lusternia.com" ), Qt::CaseInsensitive ) ) {
-
-        downloadMapOptions->setVisible( true );
+    if (pH->mUrl.contains(QStringLiteral("achaea.com"), Qt::CaseInsensitive)
+        || pH->mUrl.contains(QStringLiteral("aetolia.com"), Qt::CaseInsensitive)
+        || pH->mUrl.contains(QStringLiteral("imperian.com"), Qt::CaseInsensitive)
+        || pH->mUrl.contains(QStringLiteral("lusternia.com"), Qt::CaseInsensitive)) {
+        downloadMapOptions->setVisible(true);
```

Comment:
clang-format doesn't add blank lines to my knowledge. Is there a specific change you wanted to the `.clang-format`?